tests/pr_test.go

```diff
@@ -163,6 +163,57 @@ func TestRunUpgradeRoksPattern(t *testing.T) {
 	}
 }
 
+func TestValidateOverrideRoks(t *testing.T) {
+	t.Parallel()
+
+	options := setupOptionsRoksPattern(t, "slz-ovr-vld")
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+
+	override_json := filepath.Join(roksPatternTerraformDir, "override.json")
+	config_json := options.LastTestTerraformOutputs["config"]
+	ignored_keys := []string{"resource_group"}
+
+	//Parse the json into a map
+	var override_json_map map[string]interface{}
+	err1 := json.Unmarshal([]byte(override_json), &override_json_map)
+	fmt.Println(err1)
+
+	config_json_byteArray, ok := config_json.([]byte)
+	if !ok {
+		fmt.Println("Error: Unable to convert config_json to []byte")
+		return
+	}
+
+	var config_map map[string]interface{}
+	err2 := json.Unmarshal([]byte(config_json_byteArray), &config_map)
+	fmt.Println(err2)
```

Comment:
see similar comment above, if we should handle this error and "fail and exit"?

Reply:
Changed